plexupdate-core

```diff
@@ -228,13 +228,17 @@ isNewerVersion() {
 parseVersion() {
 	if [ "${REDHAT}" = "yes" ]; then
 		cut -f2- -d- <<< "$1" | cut -f1-4 -d.
+	elif [ "${DISTRO}" = "synology" ]; then
+		cut -f2-3 -d- <<< "$1"
 	else
 		cut -f2 -d_ <<< "$1"
 	fi
 }
 
 getPlexVersion() {
```

Comment:
For readability, functions like these should generally start with the "default" option (Ubuntu/Debian) and then go on to the other cases. It's just a bit confusingly written right now because of how the `REDHAT` variable is used. I'd suggest swapping the two first cases and adding a comment (which I really should have done when I wrote the function 🤦‍♂ ), about what the `"${REDHAT}" != "yes"` does.)

Reply:
The way GitHub is rendering this is a bit confusing. The comment is about the `getPlexVersion` function, not `isNewerVersion`.

Reply:
so if I just switch the cases like so
```
	if [ "${REDHAT}" != "yes" ]; then
		dpkg-query --showformat='${Version}' --show plexmediaserver 2>/dev/null
	elif [ "${DISTRO}" = "synology" ]; then
		synopkg version "Plex Media Server" 2>/dev/null
```
I don't think the switch ever reaches the synology case because ` "${REDHAT}" != "yes" ` is true for synology devices too.

I'm not sure how best to change it.